int16 rpmIzq
int16 rpmDer
int16 posIzq
int16 posDer